bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/converter/type/AbstractTypeConverter.java

```diff
@@ -115,9 +115,9 @@ public T convertFromBinding(HmDatapoint dp) throws ConverterException {
         if (dp.getValue() == null) {
             return (T) UnDefType.NULL;
         } else if (!fromBindingValidation(dp)) {
-            String errorMessage = String.format("Can't convert %s value '%s' with %s for '%s'", dp.getType(),
-                    dp.getValue(), this.getClass().getSimpleName(), new HmDatapointInfo(dp));
-            throw new ConverterTypeException(errorMessage);
+            logger.info("Can't convert {} value '{}' with {} for '{}'", dp.getType(), dp.getValue(),
```

Comment:
Don't you think debug is a better fit?
https://www.openhab.org/docs/developer/guidelines.html#f-logging

Reply:
Good point. I changed it